### DISCUSSION TITLE
Lost access to Supabase project, please help!

### DISCUSSION THREAD

**Discussion by NyanDesigns** [supabase/supabase#20535] [Questions]
I seem to have lost access to a Supabase project I created earlier this year to use as a backend for my React application. Full Repo link as such: https://github.com/NyanDesigns/archichat-beta.
Due to inactivity over a period of time, it may have been paused, but I don't remember the account I used for the project to be able to reactivate the project. When trying to use these to connect and query the Supabase API, I'm getting "Access Denied" errors.
There is some information hard-coded into the project, such as the Project URL, https://igaptviylbapuquphyvn.supabase.co, but I'm not sure how I can recover what the account was.
I would really appreciate any help or suggestions! Losing access to this database requires migrating all the data I had stored, which would set me back significantly. Please let me know if any other details about the project would be helpful. And if anyone has any ideas for how I could restore access given the artifacts I still have, I would hugely appreciate it!

**Comment by GaryAustin1** [comment]
If you can't get to the dashboard because you don't remember you login email or your github email changed you would need to contact support.   I'm sure they will require proof you "own" the instance like your request coming from the same email or a way to confirm the email.

 support@supabase.io
